portal/tasks.py

```diff
@@ -138,6 +141,31 @@ def test_args(*args, **kwargs):
     return "{}|{}".format(",".join(args), json.dumps(kwargs))
 
 
+@celery.task(name="tasks.consume_list", ignore_result=True, soft_time_limit=4)
+def consume_list(id_list, priority):
+    try:
+        time.sleep(random.randint(1, 5))
+        logger.info("priority: {} consuming {}".format(priority, id_list))
+    except SoftTimeLimitExceeded:
+        logger.info("timed out")
+
+
+@celery.task(name="tasks.produce_list")
```

Comment:
How is this triggered? I saw it on stg-eproms but I didn't call `/test-producer-consumer`

Reply:
that is the trigger -  GET `/test-producer-consumer`